package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "grpc-web-ci",
+  "private": true,
```

Comment:
Unrelated, but the root package should never be published as it just contains metadata for the monorepo.